### ISSUE TITLE
Hash mismatch with Python Fraction

### ISSUE THREAD

**Issue by innerhippy** [scoder/quicktions#2]
Looks like there's a discrepancy with the hash function:
```
>>> from quicktions import Fraction as cyFract
>>> from fractions import Fraction as pyFract
>>> print hash(pyFract(3, 2)), hash(cyFract(3, 2))
1610645504 314159
```

This makes comparing sets of Fraction objects with different types fail

**Comment by scoder** [comment]
Ah, yes. It's using the hash function from Python 3. You are using Python 2. Can you dig it up and send a pull request?

**Comment by scoder** [comment]
Old hash function added here:
https://github.com/scoder/quicktions/commit/d48f3938d7b810aeeab1b85f4c0a7baf94ab2c21
Released as quicktions 1.2.
